src/components/image-tooltip/ImageTooltipContent.scss

```diff
@@ -0,0 +1,29 @@
+@import '../../styles/variables';
+
+.bdl-ImageTooltipContent-image {
+    display: flex;
+
+    img {
+        width: 100%;
+    }
+}
+
+.bdl-ImageTooltipContent-contentWrapper {
+    padding: 24px 32px;
+    color: $bdl-gray-80;
+    font-size: 14px;
+    background-color: $white;
+    border-bottom-right-radius: 6px;
```

Comment:
we have instances of this 6px in a few places. i suggest creating a variable for tooltip radii, so the stay in sync

Reply:
Reaching out for design for confirmation first and then if he wants 6 pixels I'll create a variable. Thanks!